front/components/assistant/conversation/ConversationViewer.tsx

```diff
@@ -117,17 +117,15 @@ const ConversationViewer = React.forwardRef<
 
     // If latest message Id has changed, scroll to bottom of conversation.
     if (lastestMessageId && latestMessageIdRef.current !== lastestMessageId) {
-      const mainTag = document.getElementById(
-        CONVERSATION_PARENT_SCROLL_DIV_ID[isInModal ? "modal" : "page"]
-      );
+      const mainTag = document.getElementById(CONVERSATION_VIEW_SCROLL_LAYOUT);
```

Comment:
Are you sure we can get rid of `isInModal`?  Isn't it for preview in AB? (it doesn't seem to be working there but it might be already broken)

Reply:
It was indeed randomly not working on prod (scroll to the bottom of the new messge) too so we can ignore 🙏